source/tokens.rs

```diff
@@ -70,6 +78,70 @@ token!(
     "The `AS` token.\n\nRepresent the alias operator, e.g. `use Foo\\Bar as Baz`."
 );
 token!(
+    ASSIGN: "=";
+    "The `ASSIGN` token.\n\nRepresent a binding of a value to a variable, e.g. `$x = 42`."
+);
+token!(
+    ASSIGN_REFERENCE: "=&";
```

Comment:
I know. I was very hesitant with this one too. This is the grammar, but it does not mean there is no error. Also, this is not in the grammar of the Zend Engine. I will remove this one.